keras_cv/layers/preprocessing/random_hue_test.py

```diff
@@ -62,7 +55,7 @@ def test_adjust_full_opposite_hue(self):
             channel_min, tf.math.reduce_min(image, axis=-1), atol=1e-5, rtol=1e-5
         )
 
-        layer = preprocessing.RandomHue(factor=(0.75, 0.75))
+        layer = preprocessing.RandomHue(factor=(0.75, 0.75), value_range=(0, 255))
```

Comment:
I think the 0.75 is no longer the max augment. Could u check the test?

Reply:
Updated to remove.